xbmc/PlayListPlayer.h

```diff
@@ -103,6 +103,11 @@ class CPlayListPlayer : public IMsgTargetCallback
    */
   int GetCurrentPlaylist() const;
 
+  /*! \brief Get currently active playlist or a default for current player
```

Comment:
nope :) if I called this in this way GetCurrentPlaylist(PLAYLIST_NONE), what would it return? I don't see any need for this cumbersome method. it is only called in 2 places. Why don't you move the logic directly to those places?